### ISSUE TITLE
Installation priority doesn't seem to be set correctly

### ISSUE THREAD

**Issue by Pirabarlen-Cheenaramen** [rsyslog/rsyslog-pkg-rhel-centos#33]
$sudo yum repolist
```
Failed to set locale, defaulting to C
Loaded plugins: priorities, update-motd, upgrade-helper
238 packages excluded due to repository priority protections
repo id                                                                  repo name                                                                                         status
amzn-main/2016.09                                                        amzn-main-Base                                                                                       5612
amzn-updates/2016.09                                                     amzn-updates-Base                                                                                    1152
v8-stable/x86_64                                                         Adiscon CentOS-6 - local packages for \x86_64                                                     866+340
repolist: 7630
```
$ sudo yum install rsyslog
```
Failed to set locale, defaulting to C
Loaded plugins: priorities, update-motd, upgrade-helper
amzn-main/2016.09                                                                                                                                          | 2.1 kB     00:00     
amzn-updates/2016.09                                                                                                                                       | 2.3 kB     00:00     
238 packages excluded due to repository priority protections
Package matching **rsyslog-5.8.10-9.26.amzn1.x86_64** already installed. Checking for update.
Nothing to do
```

#I have to actually disable all other repos to get rsyslog to install.
#Well it used to, until some  days ago, the rsyslog package broke, and gives me this
#requires error. Though that's another issue. I should be getting the rsyslog-8 without having to disable all other repos.

$ sudo yum install rsyslog --disablerepo=\* --enablerepo=v8-stable -y
```
Failed to set locale, defaulting to C
Loaded plugins: priorities, update-motd, upgrade-helper
Resolving Dependencies
--> Running transaction check
---> Package rsyslog.x86_64 0:8.26.0-1.el6 will be updated
--> Processing Dependency: rsyslog = 8.26.0-1.el6 for package: rsyslog-gnutls-8.26.0-1.el6.x86_64
---> Package rsyslog.x86_64 0:8.26.1-1.el6 will be an update
--> Running transaction check
---> Package rsyslog-gnutls.x86_64 0:8.26.0-1.el6 will be updated
---> Package rsyslog-gnutls.x86_64 0:8.26.1-1.el6 will be an update
--> Processing Dependency: libgnutls.so.26(GNUTLS_2_12)(64bit) for package: rsyslog-gnutls-8.26.1-1.el6.x86_64
--> Finished Dependency Resolution
Error: Package: rsyslog-gnutls-8.26.1-1.el6.x86_64 (v8-stable)
           Requires: libgnutls.so.26(GNUTLS_2_12)(64bit)
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
```

**Comment by Pirabarlen-Cheenaramen** [comment]
fixed by adding
